### ISSUE TITLE
[NEW FEATURE REQUEST/IDEA]: 

### ISSUE THREAD

**Issue by inthisar-hamza** [thecodingmachine/react-native-boilerplate#420]
### Is your feature request related to a problem?

How can I add dark and light mode splash screen, is that project have this?

### Describe the solution you'd like

Dark and Light background (with or without custom light and dark brand image) according to device Theme

**Comment by captdeys** [comment]
Oh. That is a great idea! I love it. I wonder when this feature will come

**Comment by JeremyDolle** [comment]
Do you talk about the native launch screen ? For now the boilerplate doesn't implement it

**Comment by inthisar-hamza** [comment]
> Do you talk about the native launch screen ? For now the boilerplate doesn't implement it

Yeah, any ideas for future implementation? That takes this project to another level.

heavy app takes some time to load at initial launch and makes the app look unresponsive.

Anyway thanks for the work.

**Comment by JeremyDolle** [comment]
In fact the app should cold start should run quickly. Maybe you talk about StartupScreen ?
to handle splashscreen maybe you should take a look into :https://github.com/crazycodeboy/react-native-splash-screen
